polyfill/lib/ecmascript.mjs

```diff
@@ -1355,7 +1354,18 @@ function bisect(getState, left, right, lstate = getState(left), rstate = getStat
   }
   return right;
 }
-
+function unique(arr) {
+  const obj = Object.create(null);
+  for (var i = 0; i < arr.length; i++) {
+    var v = arr[i];
+    obj[v] = v;
+  }
+  var res = [];
+  for (var p in obj) {
+    res.push(obj[p]);
+  }
+  return res;
+}
```

Comment:
This is fine, but pretty heavy-handed for one call with an array of two elements. Followup to remove entirely?

Reply:
:+1:, see #488.